imap_processing/tests/hit/test_hit_l1b.py

```diff
@@ -0,0 +1,46 @@
+import pathlib
+
+import pytest
+import xarray as xr
+
+from imap_processing import imap_module_directory
+from imap_processing.cdf.utils import load_cdf
+from imap_processing.hit.l1b import hit_l1b
+
+
+@pytest.fixture(scope="module")
+def dependency():
+    """Set path to test data file"""
+
+    cdf_filepath = (
+        imap_module_directory / "tests/hit/test_data/imap_hit_l1a_hk_20100105_v001.cdf"
+    )
+    l1a_data = load_cdf(cdf_filepath)
```

Comment:
```suggestion
    pkts_filepath = (
        imap_module_directory / "tests/hit/test_data/..."
    )
    l1a_data = load_cdf(l1a_function(pkts_filepath))
```

Reply:
This is much better, thanks for the suggestion. Code has been updated